website/components/remote-plugin-docs/utils/resolve-nav-data.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs')
+const path = require('path')
+const mergeRemotePlugins = require('./merge-remote-plugins')
+const validateFilePaths = require('@hashicorp/react-docs-sidenav/utils/validate-file-paths')
+const validateRouteStructure = require('@hashicorp/react-docs-sidenav/utils/validate-route-structure')
+
+/**
+ * Resolves nav-data from file, including optional
+ * resolution of remote plugin docs entries
+ *
+ * @param {string} navDataFile path to the nav-data.json file, relative to the cwd. Example: "data/docs-nav-data.json".
+ * @param {string} localContentDir path to the content root, relative to the cwd. Example: "content/docs".
+ * @param {object} options optional configuration object
+ * @param {string} options.isDev if true, then will NOT throw errors if remote fetches fail
```

Comment:
nit: if this option only has the one use case, it might be more semantic to rename it to something like `ignoreFetchError`:

```js
resolveNavData(foo, bar, { ignoreFetchError: process.env.NODE_ENV !== 'production' })

Reply:
This does feel much clearer, updated ✅